.github/stale.yml

```diff
@@ -0,0 +1,38 @@
+# Number of days of inactivity before an Issue becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 30
+
+# Issues with these labels will never be considered stale. Set to `[]` to disable
+# We don't close any issue that is an enhancement or confirmed bug, but issues
+# waiting for reproduction cases and questions tend to get outdated.
+exemptLabels:
+  - "enhancement"
+  - "bug"
+  - "thinking"
+  - "docs"
+
+# Label to use when marking as stale
+staleLabel: "waiting-reply"
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: |
+  Hey there,
+  We wanted to check in on this request since it has been inactive for 90 days. 
```

Comment:
should we say "at least 90 days"? some will be way older initially and who knows how tight the timing is? Not a big deal though.